### PR TITLE
Set the maximum label count to 15

### PR DESCRIPTION
**Pull request by agarwalayush**
Fixes the issue #933

### CODE REVIEW

File: ilastik/applets/labeling/labelingGui.py [hunk 1]
```diff
@@ -579,7 +579,7 @@ def _updateLabelList(self):
             self._labelControlUi.labelListModel[i].name = n
                 
         if hasattr(self._labelControlUi, "AddLabelButton"):
-            self._labelControlUi.AddLabelButton.setEnabled(numLabels < self.maxLabelNumber)
+            self._labelControlUi.AddLabelButton.setEnabled(numLabels < 15)
```
Comment (burgerdev) [inline ilastik/applets/labeling/labelingGui.py]: I would prefer setting `self._maxLabelNumber = 15` over a [magic number](http://en.wikipedia.org/wiki/Magic_number_%28programming%29#Unnamed_numerical_constants) here.

**Comment by agarwalayush** [comment]
okk

**Comment by agarwalayush** [comment]
Please review and merge

**Comment by stuarteberg** [comment]
Sorry, I can't merge this for multiple reasons.

- This breaks the `@maxLabelNumber.setter` function
- The bug referenced here (#933) pertains specifically to object classification.  The right place to fix it is in the `objectClassificationGui.py`.  This change would affect other workflows, like pixel classification.
- Even if we did keep the change in `labelingGui.py`, I'd want to see a comment explaining why the number 15 was chosen.

**Comment by agarwalayush** [comment]
ok. I will find the original problem rather than quick workaround, and will send the pull request soon.

**Pull request closed without merging**